README.md

```diff
@@ -1,53 +1,77 @@
-# Role Name
+# mssql (WIP)
```

Comment:
you don't need to have (WIP) here - If you are worried that someone will accidentally merge this PR, you can add `WIP -` to the PR title

Reply:
Done.